doc/jvm/xgboost4j_spark_tutorial.rst

```diff
@@ -153,6 +153,54 @@ Now, we have a DataFrame containing only two columns, "features" which contains
 "sepal length", "sepal width", "petal length" and "petal width" and "classIndex" which has Double-typed
 labels. A DataFrame like this (containing vector-represented features and numeric labels) can be fed to XGBoost4J-Spark's training engine directly.
 
+Dealing with missing values (NaN / Null)
+--------------------------
+
+In case a feature (column) contains missing values (NaN / NULLs) for any reason (could be related to business logic / wrong data ingestion process / etc.), one should decide on a strategy of how to handle it.
+
+Possible ways are - 
+1. One can ignore it in the VectorAssembler (using setHandleInvalid = "skip"); 
+2. keep it (using setHandleInvalid = "keep"), and then XGBoost will handle it.
+
+In case a user chose strategy #2, it should handle it in the below manner:
+
+2.1. Set some irregular value to that feature.
```

Comment:
I think it's still some kind of inaccurate, 

for NaN, even users decide to keep it in VectorAssembler stage, they can set missing parameter as NaN (which is the default) to ignore it instead of transforming

only for NULL, users have to transform the dataset

Reply:
@CodingCat , we didn't talk about NaN during our discussion so I would appreciate you shed more light and then I will update it accordingly. Definitely willing to do any change so everything will be accurate here.

If you'd like, I can add the below for the "dealing with Null" section ->
****
"Another case is NaN (Not-a-Number). In case users decide to keep those in the VectorAssembler stage (using "setHandleInvalid(keep)), they will need to set the "missing" parameter as NaN (and, it is the default), so XGBoost will handle it as the missing logic, and won't train nor transform it.

The case where users need to deal with changing the values to something else is relevant mostly for Null."
****
What do you think @CodingCat ?

Just let me know and I'll add this as well, and do let me know if you think we need more cycles here.

Thanks,
Daniel